src/request_signed_tokens_request.cc

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "request_signed_tokens_request.h"
+#include "ads_serve_helper.h"
+#include "string_helper.h"
+#include "security_helper.h"
+
+#include "base/logging.h"
+#include "base/json/json_writer.h"
+#include "base/values.h"
+
+namespace confirmations {
+
+RequestSignedTokensRequest::RequestSignedTokensRequest() = default;
```

Comment:
see above builder naming convention

Reply:
Same as above, the API requests a signed token